src/node.js

```diff
@@ -558,8 +558,13 @@
              'return require("vm").runInThisContext(' +
              JSON.stringify(body) + ', { filename: ' +
              JSON.stringify(name) + ' });\n';
-    var result = module._compile(script, name + '-wrapper');
-    if (process._print_eval) console.log(result);
+    // Defer evaluation for a tick.  This is something of a workaround
+    // for deferrend events not firing when evaluating scripts from the
```

Comment:
s/deferred/ ?

Also "something of" may be better understood as "somewhat of", at least to me.